libraries/ms-helper/src/mcp2515.c

```diff
@@ -110,6 +111,10 @@ static void prv_handle_error(Mcp2515Storage *storage, uint8_t int_flags) {
     prv_write(storage, MCP2515_CTRL_REG_EFLG, &clear, 1);
     prv_write(storage, MCP2515_CTRL_REG_TEC, &clear, 1);
     prv_write(storage, MCP2515_CTRL_REG_REC, &clear, 1);
+
+    // Why are we doing this?
```

Comment:
We should probably remove this comment and document what's happening here

Reply:
Yea I'm still updating the comments and I don't know why we're doing that